tflib/helm-cleanup/main.tf

```diff
@@ -0,0 +1,13 @@
+terraform {
+  required_providers {
+    oci = { source = "chainguard-dev/oci" }
+  }
+}
+
+variable "name" {}
+
+data "oci_exec_test" "cleanup" {
```

Comment:
If you just want to run a script, you can use `null_resource`:

https://registry.terraform.io/providers/hashicorp/null/latest/docs/resources/resource
https://developer.hashicorp.com/terraform/language/resources/provisioners/local-exec

```
resource "null_resource" "cleanup" {
  provisioner "local-exec" {
    command = "helm uninstall ${var.name} --wait"
  }
}
```

Reply:
neat lemme try that